src/ploigos_step_runner/__init__.py

```diff
@@ -46,6 +46,7 @@
 * runtime-vulnerability-scan
 * canary-test
 * publish-workflow-results
+* automated-governance
```

Comment:
please also add example config to the example config file blocks in this file for automated-governance/rekor.